proto/storage/compliance_operator_v2.proto

```diff
@@ -135,6 +135,24 @@ message ComplianceOperatorClusterScanConfigStatus {
   string cluster_name = 5;
 }
 
+// Next Tag: 8
+message ComplianceOperatorBenchmarkV2 {
+  string id = 1; // @gotags: sql:"pk,id,type(uuid)"
+  string name = 2; // @gotags: search:"Compliance Benchmark Name,hidden"
+  string version = 3;
+  string description = 4;
+  // Example: https://control.compliance.openshift.io/CIS-OCP
+  repeated string profile_annotation = 5;
```

Comment:
I think this needs to go into the Profile section and would make life much simpler if it were searchable.

Reply:
Question, could we have multiple profile annotation per profile or something like this will work?

```proto
  message Profile {
    string profile_name = 1; // @gotags: search:"Compliance Profile Name" sql:"fk(ComplianceOperatorProfileV2:name),no-fk-constraint"
    string profile_version = 2; // @gotags: search:"Compliance Profile Version" sql:"fk(ComplianceOperatorProfileV2:profile_version),no-fk-constraint"
    // Example: https://control.compliance.openshift.io/CIS-OCP
    string profile_annotation = 3;
  }

  repeated Profile profiles = 6;
```

Reply:
Yes this should be what we want. The same annotation may occur in multiple profiles, but each profile should only have 1 annotation.  

Life will be simpler if we make it searchable so we can return it in a query though.
```
string profile_annotation = 3 // @gotags: search:"Compliance Profile Annotation";
```

Reply:
> `@gotags: search:"Compliance Profile Annotation";` 

🤦 forgot the gotag. Thanks for the reminder 🙂